src/main/java/sg/dex/starfish/keeper/TokenManager.java

```diff
@@ -0,0 +1,19 @@
+package sg.dex.starfish.keeper;
+
+import com.oceanprotocol.squid.exceptions.TokenApproveException;
+import com.oceanprotocol.squid.manager.BaseManager;
+import com.oceanprotocol.common.web3.KeeperService;
+
+public class TokenManager extends BaseManager {
```

Comment:
Why do we need this class at all? Why not just use the token  functionality in Squid? Comments in code required.

Reply:
Done. all commented